ee/clickhouse/client.py

```diff
@@ -114,6 +118,8 @@ def sync_execute(query, args=None, settings=None):
             if app_settings.SHELL_PLUS_PRINT_SQL:
                 print(format_sql(query, args))
                 print("Execution time: %.6fs" % (execution_time,))
+                g = statsd.Gauge("%s_clickhouse_sync_execution_time" % (settings.STATSD_PREFIX,))
+                g.send("clickhouse_sync_query_time", execution_time)
```

Comment:
This only works locally - in production `SHELL_PLUS_PRINT_SQL` is False

Reply:
🥴